src/i18n/en.json

```diff
@@ -277,11 +471,39 @@
         },
         "fumbleRoll": "Roll Fumble",
         "hitLocationRoll": "Roll Hit location",
-        "Modifier": "Modifier"
+        "Modifier": "Modifier",
+        "CantFindSkillError": "Can't find skill with itemId [{skillId}] on actor {actorName} to show a weapon chat card.",
+        "CantFindActorWarn": "Can't find world actor to do action",
+        "UsedLastOfAmmoWarn": "You just used the last of your {projectileName}!",
+        "OutOfAmmoWarn": "You don't have any {projectileName} to {combatManeuverName}!",
+        "CantFindChatMessageError": "Can't find chat message",
+        "HitLocationRollFlavor": "Hit Location Roll",
+        "UnkownButtonInCardError": "Unknown button \"{actionButton}\" in weapon chat card",
```

Comment:
```suggestion
        "UnknownButtonInCardError": "Unknown button \"{actionButton}\" in weapon chat card",
```

Reply:
I'm going to fix the spelling in the key, but I'm not actually finding the key anywhere.

Reply:
Nevermind, found it!